pages/am/3.x/dev-guide/protocols/saml2/protocols_saml2_configuration.adoc

```diff
@@ -0,0 +1,67 @@
+= Enable SAML 2.0 Identity Provider support
+:page-sidebar: am_3_x_sidebar
+:page-permalink: am/current/am_devguide_protocols_saml2_configuration.html
+:page-folder: am/dev-guide/protocols/saml2
+:page-layout: am
+
+[label label-enterprise]#Enterprise feature#
+
+[label label-version]#Early stage#
+
+[label label-version]#New in 3.17.0#
+
+== Supported SAML bindings and options
+
+Currently AM supports the following SAML bindings :
+
+- HTTP-Redirect
+
+NOTE: HTTP-POST binding is planned for a next future of the SAML 2.0 IdP protocol plugin.
+
+and the following options :
+
+- Web Browser SSO Profile
+- Single Logout Profile
+- SP-Initiated flow
+- Support for signed SAML assertions (SAML Request and SAML Response)
+
+NOTE: Support for encrypted SAML assertions will be provided in a next version of the SAML 2.0 IdP protocol plugin.
+
+== Activate SAML 2.0 IdP
+
+NOTE: Be sure to have your SAML 2.0 IdP protocol plugin and your license key installed in your environment before configuring the connection.
```

Comment:
Do we have to specify on which service ?
Currently only the GW need such installation.

Reply:
I don't know to be honest